src/Controller/CapturedEmailController.php

```diff
@@ -20,7 +27,7 @@ public function view() {
 		$id = (int) $this->getRequest()->param('ID');
 
 		if ($id) {
-			$email = DataList::create('CapturedEmail')->byID($id);
+			$email = DataList::create(CapturedEmail::class)->byID($id);
```

Comment:
Preferable to use `CapturedEmail::get()->byID($id);`

Reply:
Yeah totally, not sure what the original intent there was, this was part of the raw upgrade so I just 'made it work'.